anathema-widgets/src/widget/attributes.rs

```diff
@@ -200,6 +200,15 @@ impl CellAttributes for Attributes<'_> {
         }
     }
 
+    fn get_color(&self, key: &str) -> Option<anathema_state::Color> {
+        let value = self.get_val(key)?;
+        let value = value.load_common_val()?;
+        match value.to_common()? {
+            CommonVal::Color(color) => Some(color),
```

Comment:
```suggestion
            CommonVal::Color(color) => Some(color),
            CommonVal::Str(str) => Color::from_str(str).ok(),
            CommonVal::Int(ansi_value @ 0..=u8::MAX as i64) => Some(Color::AnsiVal(ansi_value as u8)),
```

The string case is handled by the TUI. I would suggest moving it down here with the code above. This way, lines 143-152 of `anathema-backend/src/tui/style.rs` can be simplified.